### DISCUSSION TITLE
"Page not found" after clicking a row, but no error showed up in console.

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#10012] [Archive]
My operation steps:

1. Change a table's PK from column A to column B, where A's `directus_fields.special = uuid`, and B is also a FK.
2. Drop column A.
3. Remove all records related to column A in Directus meta tables, like `directus_fields`.
4. Restart Directus.
5. In Directus GUI, navigate to the table and click on one row.
6. The "Page not found" error shows up, but no error in console where Directus is running.

Could anyone give me some hints about where could go wrong?

Thanks!

**Comment by rijkvanzanten** [comment]
No clue! We don't currently support changing the primary key from the platform, so it's a little hard for us to figure out what's going wrong once you start modifying the database directly. The only thing I can think of is that you have to make sure to clear the caches (or restart the server, which also flushes the caches) when you do any modifications like that straight on the database with schema caching enabled.
